packages/react/src/components/filterBox/FilterBox.tsx

```diff
@@ -82,12 +88,25 @@ export class FilterBox extends Component<IFilterBoxProps, any> {
 
     static defaultProps: Partial<IFilterBoxProps> = {
         isAutoFocus: false,
+        disabled: false,
+    };
+
+    private handleIcons = (inputValue: string) => {
+        if (inputValue.length !== 0) {
+            this.filterInput.nextElementSibling.setAttribute('class', ' flex filter-box-icon hidden'); // hide filter icon
+            this.filterInput.nextElementSibling.nextElementSibling.classList.remove('hidden'); // show clear icon
+        } else {
+            this.filterInput.nextElementSibling.classList.remove('hidden'); // show filter icon
+            this.filterInput.nextElementSibling.nextElementSibling.setAttribute(
+                'class',
+                ' flex filter-box-icon hidden'
+            ); // hide clear icon
```

Comment:
Instead of relying on nextSibling would it be possible to use refs?

Reply:
Probably but since the svg is getting kicked fairly soon, I would prefer not to spend too much time on the icons but I take note for when I merge this in next! 👍